### PR TITLE
math: remove unused code inside of unittest block

### PR DESCRIPTION
**Pull request by ljmf00**
This code affects code coverage and shouldn't be here at all.

Signed-off-by: Luís Ferreira <contact@lsferreira.net>

### CODE REVIEW

**Comment by thewilsonator** [comment]
I presume that  is there to aid debugging, could you pease wrap that in `version(none)` instead.

**Comment by ljmf00** [comment]
> I presume that is there to aid debugging, could you pease wrap that in `version(none)` instead.

Force pushed :smile: . Can you check now?

**Review by thewilsonator — APPROVED** [review]

File: std/internal/math/biguintcore.d [hunk 1]
```diff
@@ -2871,18 +2871,21 @@ pure nothrow @safe
 
 @system unittest
 {
-    import core.stdc.stdio;
-
-    void printBiguint(const uint [] data)
+    version(none)
```
Comment (thewilsonator) [inline std/internal/math/biguintcore.d:2874]:
```
Enforce whitespace before opening parenthesis
  | grep -nrE "\<(for\|foreach\|foreach_reverse\|if\|while\|switch\|catch\|version)\(" $(find etc std -name '*.d') ; test $? -eq 1
  | std/internal/math/biguintcore.d:2874:    version(none)
 ```

This needs to be `version (none)`

Comment (ljmf00) [reply std/internal/math/biguintcore.d]: Amended. Sorry for the inconvenience.

**Pull request merged**